LearningJournal/tests/test_LJAPI.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+"""Test get json."""
+import os
+
+from .conftest import dbtransaction, loaded_db
+from sqlalchemy import engine_from_config
+from sqlalchemy.orm import scoped_session, sessionmaker
+from zope.sqlalchemy import ZopeTransactionExtension
+
+
+def test_get_json():
+    """Test to make sure we are getting correct data from API."""
+    from LJAPI import get_json
+    test = get_json()
+    for entry in test:
+        if "David Flegal" in entry['title']:
+            return entry
+        else:
+            pass
+    assert "David Flegal" in entry['title']
```

Comment:
This test doesn't do anything.  Getting the JSON is the least interesting part of this process.  Can you add some tests that demonstrate what you do with the entries *after* they've been retrieved?